wlp-managed-8.5/src/main/java/org/jboss/arquillian/container/was/wlp_managed_8_5/WLPManagedContainer.java

```diff
@@ -576,10 +707,23 @@ private Element createApplication(Document doc, String deploymentName, String ar
          application.appendChild(sharedLib);
       }
 
+      if(containerConfiguration.getSecurityConfiguration() != null) {
```

Comment:
I think the account you opened the JIRA ticket with should also be able to make edits to that link.